ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/contentfragment/v1/contentfragment/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    componentGroup="CIF Core Components"
+    cq:icon="tableHistogram"
+    jcr:primaryType="cq:Component"
+    jcr:title="Commerce Content Fragment"
```

Comment:
Lets add `Placeholder` to the title, to follow same pattern we have for CIF XF placeholder component.

Reply:
I've removed `Placeholder' for consistency from XF component as we agreed.